infrastructure/modules/monitoring/fluent-bit/configs/filter/filter-kubernetes-daemonset.conf

```diff
@@ -1,6 +1,6 @@
 [FILTER]
     Name                kubernetes
-    Match               kube.*
+    Match               *
```

Comment:
Could you explain why this match has been relaxed?

Reply:
To use the same filte for seq and cloudwatch